src/electionguard/serialize.hpp

```diff
@@ -1290,12 +1292,15 @@ namespace electionguard
 
                 auto nonce = ballot_nonce.empty() ? make_unique<ElementModQ>(ZERO_MOD_Q())
                                                   : ElementModQ::fromHex(ballot_nonce);
+                auto state = serialized_state == 0
```

Comment:
Do we want to change the default value of unknown to 0 instead of 999?  This way we would not need this logic.

Reply:
https://github.com/microsoft/electionguard-core2/issues/165